sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,192 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+This module requires llvmlite (https://github.com/numba/llvmlite).
+'''
+
+import ctypes
+
+from sympy.external import import_module
+from sympy.printing.printer import Printer
+from sympy import S
+from sympy.utilities.decorator import doctest_depends_on
+
+llvmlite = import_module('llvmlite')
+if llvmlite:
+    ll = import_module('llvmlite.ir').ir
+    llvm = import_module('llvmlite.binding').binding
+    llvm.initialize()
+    llvm.initialize_native_target()
+    llvm.initialize_native_asmprinter()
+
+
+class LLVMJitPrinter(Printer):
```

Comment:
Does llvmlite have a function to spit out the IR? If so, one could just use that (and we could document it). If not, then it's not worth it. The point of llvmlite is to have an abstraction over the IR. Writing a printer directly to it would defeat the purpose, and be a lot more work (you'd have to essentially rewrite llvmlite I'd guess).